Bio/SubsMat/FreqTable.py

```diff
@@ -38,10 +38,18 @@
   count: count dictionary. Empty if no counts are provided.
 
 Example of use:
-  >>> from SubsMat import FreqTable
-  >>> ftab = FreqTable.FreqTable(my_frequency_dictionary,FreqTable.FREQ)
-  >>> ftab = FreqTable.FreqTable(my_count_dictionary,FreqTable.COUNT)
-  >>> ftab = FreqTable.read_count(open('myDNACountFile'))
+    >>> import io
+    >>> from Bio.SubsMat import FreqTable
+    >>> f_count = io.StringIO("A  50\nC  37\nG  23\nT  58")
+    >>> ftab = FreqTable.read_count(f_count)
+    >>> nucleobases = list(ftab.keys())
+    >>> nucleobases.sort()
+    >>> for nb in nucleobases:
```

Comment:
How about replacing those three lines with:

```python
>>> for nb in sorted(ftab):
```

If we're lucky this will also fix the Python 2 problem which I have not looked closely at.